### PR TITLE
File workbench trans strings, #8656

### PR DESCRIPTION
**Pull request by njkim**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Issues Solved

#8656 

### Checklist

-   [ ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @

### CODE REVIEW

File: arches/app/templates/views/components/file-workbench.htm [hunk 1]
```diff
@@ -162,27 +176,32 @@ <h4 class="r-select-title" data-bind="text: $data.title"></h4>
...
-                       <div style="padding: 10px">{% trans "Adding files to this record is " %}
-                           <span><strong>{% trans "optional." %}</strong></span>
-                           <br>
-                           {% trans "You may upload as many files as you wish; check with the site admin on the maximum file size." %}
-                       </div>
+                            <h2>
+                                <span data-bind="text: translations.uploadFiles"></span>
+                            </h2>
+                            <h4>
+                                <span data-bind="text: translations.dropzone-file-upload-caption"></span>
```
Comment (chrabyrd) [inline arches/app/templates/views/components/file-workbench.htm:183]: calls to `translations` should be camel cased

**Review by chrabyrd — CHANGES_REQUESTED** [review]
overall good just 1 thing

File: arches/app/templates/views/components/file-workbench.htm (hunk 1 above)
Comment (njkim) [reply arches/app/templates/views/components/file-workbench.htm:183]: Thanks.

**Review by chrabyrd — APPROVED** [review]
looks good!

**Pull request merged**